docs/operating-scylla/procedures/cluster-management/arbiter-dc.rst

```diff
@@ -0,0 +1,36 @@
+=======================================================
+Preventing Quorum Loss in Multi-DC Clusters
+=======================================================
+
+ScyllaDB requires at least a quorum (majority) of nodes in a cluster to be
+available. If a quorum is lost, schema and topology updates are impossible.
+
+In two-datacenter (2-DC) clusters that are symmetrical, i.e., have the same
+number of nodes, losing a quorum may occur if one of the DCs becomes unavailable.
+For example, if one DC fails in a 2-DC cluster where each DC has three nodes,
+only three out of six nodes are available, and the quorum is lost.
+
+Adding another DC would mitigate the risk of losing a quorum, but it comes
+with network and storage costs. To prevent the quorum loss with minimum costs,
+you can configure an arbiter (tie-breaker) DC.
+
+An arbiter DC is a datacenter where all nodes are zero-token nodes -- nodes
```

Comment:
What is the advantage of having more than one zero-node token node in the Arbiter DC?

Reply:
I do not see an advatage for using more than one zero-token node in an Arbiter DC

Reply:
> What is the advantage of having more than one zero-node token node in the Arbiter DC?

Having more than one zero-token node in the Arbiter DC in the symmetrical 2-DC setup has some theoretical advantage. It can prevent quorum loss in more extreme scenarios. For example, with three zero-token nodes you don't lose quorum if the whole DC1 dies and one node in DC2 dies. However, I think we shouldn't consider such cases in the documentation. It will only complicate everything. I would keep it simple: one goal -- prevent quorum loss when one DC dies, one solution -- the Arbiter DC with one zero-token node.

> Me neither, but the resources I used for reference mentioned nodes and highlighted the fact that if there are multiple nodes, they all must be zero-token nodes.

In general, you can have any number of zero-token nodes and any number of token-owning nodes in any DC. The only constraint is that there must be at least one token-owning node in the whole cluster. So, a DC with > 0 token-owning nodes and > 0 zero-token nodes will work correctly.

However, adding such a DC for the purpose of preventing quorum loss doesn't make sense. It's overcomplicated and not efficient. We recommend using zero-token nodes (instead of normal token-owning nodes that would also work) in the Arbiter DC because:
- zero-token nodes don't replicate data,
- drivers ignore zero-token nodes,

which makes zero-token nodes:

- cheaper (they can run on much weaker machines because they don't store user data and don't handle client requests),
- simpler to use and reason about (no need to change the driver load balancing policies, no impact on token the ring, etc.).

Anyway, I still think we can keep it simple -- the Arbiter DC contains one zero-token and that's it. If someone adds a DC with more than one node or with a token-owning node, then they don't add the Arbiter DC. They do something outside the scope of this documentation.